assets/app/view/game/map.rb

```diff
@@ -47,7 +48,12 @@ def render
           left = (@tile_selector.x + map_x) * SCALE
 
           selector =
-            if @tile_selector.hex.tile != @tile_selector.tile
+            if @tile_selector.is_a? Lib::TokenSelector
+              # 1882
+              h(TokenSelector,
```

Comment:
you don't need to pass in game because it's store true, maybe we should change to just pass in the selector

h(TokenSelector, tile_selector: @tile_snelector)